samples/web/CMakeLists.txt

```diff
@@ -84,7 +84,7 @@ function(add_ktxfiles SOURCE TARGET EXTRA_ARGS)
 endfunction()
 
 add_ktxfiles("assets/models/monkey/albedo.png" "monkey/albedo.ktx" "")
-add_ktxfiles("assets/models/monkey/normal.png" "monkey/normal.ktx" "--kernel=NORMALS")
+add_ktxfiles("assets/models/monkey/normal.png" "monkey/normal.ktx" "--kernel=NORMALS;--linear")
```

Comment:
Why the ; separator? Aren't those CLI flags?

Reply:
By using semicolon, we're telling CMake that this is a list of args, so invokes the syscall correctly; when I used a space it passed "--kernel=NORMALS --linear" into a single arg.

https://llvm.org/docs/CMakePrimer.html#lists